crates/core/component/funding/src/component.rs

```diff
@@ -30,7 +30,9 @@ impl Component for Funding {
     #[instrument(name = "funding", skip(state, app_state))]
     async fn init_chain<S: StateWrite>(mut state: S, app_state: Option<&Self::AppState>) {
         match app_state {
-            None => { /* Checkpoint -- no-op */ }
+            None => {
+                state.put_funding_params(FundingParameters::default());
```

Comment:
Ah. I misspoke, and I understand your previous point better, I was thinking of the other variant `Some(app_state)`. Here this is indeed a no-op. The `None` case is only

Let me fix it